controller/src/main/java/io/pravega/controller/store/stream/AbstractStreamMetadataStore.java

```diff
@@ -460,6 +460,12 @@ public OperationContext createContext(String scope, String name) {
         return withCompletion(getStream(scope, stream, context).getActiveTxns(), executor);
     }
 
+    @Override
+    public CompletableFuture<List<ScaleIndicent>> getScaleIncidents(final String scope, final String name,
```

Comment:
Why are we calling it `incidents`? We have always called it events, so let's stick to `events` unless there is a good reason to move away from it.